### ISSUE TITLE
How can I modify the number of classes

### ISSUE THREAD

**Issue by RikkiXu** [huang-yh/SelfOcc#6]
Thank you for your code, but I encountered problems when reproducing it. How can I modify the number of classes in NeuSHead? I used forward_occ to get the logits as 200x200x16x21, but I didn't find where to modify 21. I want to modify it to 16

**Comment by huang-yh** [comment]
https://github.com/huang-yh/SelfOcc/blob/7c2247479a09fe208a4782e5a399493d92d0b61b/config/nuscenes/nuscenes_occ.py#L350
You can modify the above line to change the number of predicted classes. Note that color_dims=3(RGB) + #classes.
